src/components/Dropdown.jsx

```diff
@@ -31,4 +32,15 @@ class Dropdown extends React.Component {
 	}
 }
 
+Dropdown.propTypes = {
+	onChange: PropTypes.function,
```

Comment:
PropTypes uses `PropTypes.func`, not function, to avoid hitting that keyword. Make sure you have the terminal open, it'll complain about stuff like this to you.